policy/policy.go

```diff
@@ -335,8 +336,8 @@ func (l* Location) String() string {
 
 /** Contents are used to pass opaque plugin specific information */
 type Contents struct {
-    PluginId string
-    Blob string
+    PluginId string	    `json:"plugin_id"`
+    Blob string		    `json:"blob"`
```

Comment:
interesting question  that I don't have an answer to (and a brief search was not productive) this can, technically, be binary data how will with be output?  do we need to manually base64 it?

Reply:
Good point. I've changed the type to `[]byte` and now it is base64 encoded when serializing.